dataset/dictionary/enemies.ts

```diff
@@ -2434,22 +2443,13 @@ export default [
     pronunciationJa: "せんねんしんじゅのかいしゅん",
     tags: [ "fontaine", "enemy-boss" ],
   },
-
-  {
-    en: "local legend",
-    ja: "地方伝説",
-    zhCN: "地方传奇",
-    zhTW: "地方傳奇",
-    tags: [ "fontaine", "enemy" ],
-    notes: "フォンテーヌに点在する、強めの敵個体",
-  },
   {
     en: "Vivianne of the Lake",
     ja: "湖辺のヴィヴィアン",
     zhCN: "湖畔的维维安涅",
     zhTW: "湖畔的維維安涅",
     pronunciationJa: "こへんのヴィヴィアン",
-    tags: [ "fontaine", "enemy" ],
+    tags: [ "fontaine", "enemy", "enemy-legend" ],
```

Comment:
Looks like there are Local Legends only with `enemy-legend` tag and the ones with both `enemy` and `enemy-legend` tags.
I think we should make them consistent.

Which do you prefer, adding `enemy` tag or not?
I considered to replace `enemy` tags with `enemy-legend` tags from the Local Legends because the Bosses does not have `enemy` tag.
However, I don't have any strong opinion for this and adding both `enemy` and `enemy-legend` is also fine for me.